src/main/resources/org/jenkinsci/plugins/workflow/flow/GlobalDefaultFlowDurabilityLevel/help-durabilityHint.html

```diff
@@ -0,0 +1,20 @@
+<div>
+    <p>
+        This setting provides a trade-off between how fast pipelines run and their "durability", or ability to survive unexpected failures while running.
+
+        <br><strong>Facts:</strong>
+        <ul>
+            <li>The lowest durability modes (suggested) can be MUCH faster and write to disk FAR less often.</li>
+            <li>All pipelines persist their data when they complete/fail, are paused, or the master restarts gracefully.</li>
+            <li>Running pipelines that do NOT persist data with every step (lower durability settings) may lose data if Jenkins is not shut down gracefully.</li>
+            <li>Pipelines may not be able to resume or be visualized if they cannot persist data.</li>
+            <li>If a running pipeline loses data, you can always run the pipeline again.</li>
```

Comment:
Heh, this one isn't the most actionable or useful bit of information for an end-user. I suggest dropping it.

Reply:
So, @rtyler IIUC basically you're saying to overall just remove the help section?

Reply:
No, this bullet point in particularly.